src/components/elements/upload/fileSummary/FileRecordSummary.tsx

```diff
@@ -0,0 +1,50 @@
+import { Typography } from '@mui/material';
```

Comment:
Move to `clientFiles` module? (And maybe the module should be renamed to just `files`)